pygerduty/__init__.py

```diff
@@ -403,6 +416,11 @@ def resolve(self, requester_id):
     def acknowledge(self, requester_id):
         self._do_action('acknowledge', requester_id=requester_id)
 
+    def get_details_log_entry(self, requester_id):
```

Comment:
I'm not certain about this method name. This specifically grabs the single log entry related to triggering the incident. You can currently list all log_entries with the channel like so

`incident.log_entries.list(include=["channel"])`

If you're looking specifically for the "trigger_log_entry" it's worth being more explicit with the name. If that's the case then you can use the id from `trigger_details_html_url` with `self.log_entries.show(some_id, include=["channel"])` instead of modifying the `_do_action` method. That should further simplify your actual pull request.